src/plasmapy/plasma/cylindrical_equilibria.py

```diff
@@ -51,6 +58,16 @@ def B_theta(self, r):
         Returns
         -------
         `~astropy.units.Quantity`
+
+        Examples
+        --------
+        >>> import astropy.units as u
+        >>> B0 = 1 * u.T
+        >>> alpha = 1 * 1 / u.m
+        >>> fr = ForceFreeFluxRope(B0, alpha)
+        >>> r = [5, 3, 9] * u.m
+        >>> fr.B_theta(r)
+        <Quantity [-0.32757914, 0.33905896, 0.24531179] T>
```

Comment:
```suggestion
        >>> r = [3, 5, 9] * u.m
        >>> fr.B_theta(r)
        <Quantity [0.33905896, -0.32757914, 0.24531179] T>
```
...since I suspect that for most applications the radii will be monotonically increasing or decreasing.